EpikCord/client/event_handler.py

```diff
@@ -160,112 +115,8 @@ async def dispatch(self, event_name: str, *args, **kwargs):
-
-        if guild:
-            await self.dispatch("guild_delete", guild)
-
-    async def _interaction_create(self, data: discord_typings.InteractionCreateEvent):
-        interaction = self.utils.interaction_from_type(data)
-        await self.handle_interaction(interaction)
-        return interaction
-
-    async def _channel_create(self, data: discord_typings.ChannelCreateEvent):
-        channel = self.utils.channel_from_type(data)  # type: ignore
-        self.channels.add_to_cache(channel.id, channel)  # type: ignore
-        return channel
-
-    async def _message_create(self, data: discord_typings.MessageCreateEvent):
-        """Event fired when messages are created"""
-        from EpikCord import Message
-
-        message = Message(self, data)
-        message.channel = self.channels.get(  # type: ignore
-            message.channel_id
-        ) or await self.channels.fetch(
-            message.channel_id
-        )  # type: ignore
-        message.channel.last_message = message
-
-        return message
-
-    async def _guild_create(self, data: GuildCreateEvent):
-        from EpikCord import Guild, Thread, UnavailableGuild
-
-        if data.get("unavailable") is None:
-            return  # TODO: Maybe a different event where the name says the Bot is removed on startup.
-
-        guild = (
-            UnavailableGuild(data)
-            if data.get("unavailable") is True
-            else Guild(self, data)
-        )
-
-        if not guild:
-            return
-
-        self.guilds.add_to_cache(guild.id, guild)
-
-        for channel in data["channels"]:
-            self.channels.add_to_cache(
-                data["id"], self.utils.channel_from_type(channel)
-            )
-
-        for thread in data["threads"]:
-            self.channels.add_to_cache(data["id"], Thread(self, thread))
-
-        return guild
-
-        # TODO: Add other attributes to cache
-
-    async def _guild_member_update(self, data: GuildMemberUpdateEvent):
-        from EpikCord import GuildMember
-
-        guild_member = GuildMember(self, data)
-        return self.members.get(data["id"]), guild_member
-
-    async def _ready(self, data: discord_typings.ReadyEvent):
-        from EpikCord import ClientApplication, ClientUser
-
-        self.user: ClientUser = ClientUser(self, data["user"])
-        self.session_id: Optional[str] = data["session_id"]
-        application_response = await self.http.get("/oauth2/applications/@me")  # type: ignore
-        application_data = await application_response.json()
-
-        self.application = ClientApplication(self, application_data)
-
-        if not self.overwrite_commands_on_ready:  # type: ignore
-            return
-
-        await self.utils.overwrite_commands()  # type: ignore
-        await self.dispatch("ready")
-
     async def command_error(self, interaction, error: Exception):
         logger.exception(error)
 
 
-__all__ = ("Event", "EventHandler")
+__all__ = ("Event", "EventHandler")
```

Comment:
`Event` is deleted in code, remove that from `__all__` aswell.

Reply:
Yep, good spot.